### PR TITLE
Edits to Feedback language

### PR DESCRIPTION
**Pull request by jeremyzilar**
**Based on the discussion here:** https://github.com/GSA/digitalgov-pra/issues/219

This addresses the first two items in this comment: https://github.com/GSA/digitalgov-pra/issues/219#issuecomment-499882551

- Update the language in the current header to say "email us feedback" and go to a mailto link directly.
- Update the feedback text in the footer

**Preview:** https://cg-06ab120d-836f-49a2-bc22-9dfb1585c3c6.app.cloud.gov/preview/gsa/digitalgov-pra/feedback-edits/

### CODE REVIEW

**Comment by cmajel** [comment]
@jeremyzilar it looks like there are two different templates being used for the feedback url.

 For the top url
Screen Shot 2019-06-12 at 10 31 06 AM

For the bottom url
Screen Shot 2019-06-12 at 10 34 47 AM

Could we use the second version universally? The subject line is more specific (feedback on the guide vs feedback on the PRA) and also provides a few prompts in the email body.

We should also confirm with @qhirschomb or @mmcmanusOMB on our final decision so that they can triage these messages effectively / know what subject to watch for.

File: _includes/demo.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <div class="grid-container grid-container-desktop">
     <div class="grid-row grid-gap-2">
       <div class="grid-col-12">
-        <p><strong>This is a preview of the <em>new</em> Guide to the PRA.</strong> <a href="{{ '/about#feedback' | relative_url }}"><span>We're looking for your feedback</span> »</a></p>
+        <p><strong>This is a preview of the <em>new</em> Guide to the PRA.</strong> <a href="mailto:PRA@omb.eop.gov?subject=Feedback%20on%20PRA"><span>Email us your feedback</span> »</a></p>
```
Comment (hursey013) [inline _includes/demo.html]: I included a way to obfuscate the email address to help reduce spam: https://github.com/GSA/digitalgov-pra/wiki/Adding-a-link-to-the-site-feedback-email-address

Comment (jeremyzilar) [reply _includes/demo.html]: Oh nice! thank you @hursey013 🎉

Comment (ecayer) [reply _includes/demo.html]: Could I just suggest changing "Feedback on PRA" to "Feedback on the PRA Guide"?

**Pull request merged**